From 36e2b2b2f01ec9c601d5530d1b9330e76f158437 Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Mon, 5 Feb 2018 15:19:38 +0800
Subject: [PATCH] arm: dts: fix reg length err for rk322x

RK322x is a 32-bit SoC, the length of register is 32-bit.

Change-Id: I3fe506b0f9c7f8ec70a56863f95decdc12e61e85
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm/boot/dts/rk322x.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/rk322x.dtsi b/arch/arm/boot/dts/rk322x.dtsi
index 772b61795238..153babf5e393 100644
--- a/arch/arm/boot/dts/rk322x.dtsi
+++ b/arch/arm/boot/dts/rk322x.dtsi
@@ -1043,17 +1043,17 @@
 
 	qos_vpu: qos@31040000 {
 		compatible = "syscon";
-		reg = <0x0 0x31040000 0x0 0x20>;
+		reg = <0x31040000 0x20>;
 	};
 
 	qos_rkvdec_r: qos@31070000 {
 		compatible = "syscon";
-		reg = <0x0 0x31070000 0x0 0x20>;
+		reg = <0x31070000 0x20>;
 	};
 
 	qos_rkvdec_w: qos@31070080 {
 		compatible = "syscon";
-		reg = <0x0 0x31070080 0x0 0x20>;
+		reg = <0x31070080 0x20>;
 	};
 
 	gic: interrupt-controller@32010000 {
-- 
2.35.3

